README.md

```diff
@@ -1,229 +1,67 @@
-```
-./azcopy cp /data/directory1/* "https://myaccount.blob.core.windows.net/mycontainer/directory1?sastokenhere"
-```
+:white_check_mark: Download files and folders.
 
-To upload into File storage, simply change the URI to Azure File URI with corresponding SAS token.
+:white_check_mark: Copy containers, folders and blobs between storage accounts (Blobs only).
 
-### Copy VHD image to Azure Storage
+:white_check_mark: Synchronize containers with local file systems and visa versa (Blobs only).
 
-AzCopy by default uploads data into Block Blobs. However if a source file has `.vhd` extension, AzCopy will default to uploading to a Page Blob. 
+:white_check_mark: Copy objects, folders, and buckets from Amazon Web Services (AWS) (Blobs only).
 
-### Copy data from Azure to local file systems
+:white_check_mark: List files in a container (Blobs only).
 
-The following will download all Blob container contents into the local file system creating the directory `mycontainer` in the destination.
-```
-./azcopy cp "https://myaccount.blob.core.windows.net/mycontainer?sastokenhere" /data/ --recursive=true
-```
+:white_check_mark: Remove files from a container (Blobs only).
 
-The following will download all Blob container contents into the local file system. `mycontainer` directory will not be created in the destination because the globbing pattern looks for all paths inside `mycontainer` in the source rather than the `mycontainer` container itself.
-```
-./azcopy cp "https://myaccount.blob.core.windows.net/mycontainer/*?sastokenhere" /data/ --recursive=true
-```
+:white_check_mark: Recover from failures by restarting previous jobs.
 
-The following command will download all txt files in the source to the `directory1` path. Note that AzCopy will scan the entire source and filter for `.txt` files. This may take a while when you have thousands/millions of files in the source.
-```
-./azcopy cp "https://myaccount.blob.core.windows.net/mycontainer/directory1/*.txt?sastokenhere" /data/directory1
-```
+## Find help
 
-### Copy data between Azure Storage accounts (currently supports Block Blobs only)
+For complete guidance, visit any of these articles on the docs.microsoft.com website.
 
-Copying data between two Azure Storage accounts make use of the PutBlockFromURL API, and does not use the client machine's network bandwidth. Data is copied between two Azure Storage servers. AzCopy simply orchestrates the copy operation.
-```
-./azcopy cp "https://myaccount.blob.core.windows.net/?sastokenhere" "https://myotheraccount.blob.core.windows.net/?sastokenhere" --recursive=true
-```
+:eight_spoked_asterisk: [Get started with AzCopy](https://docs.microsoft.com/azure/storage/common/storage-use-azcopy-v10)
 
-### Advanced Use Cases
+:eight_spoked_asterisk: [Transfer data with AzCopy and blob storage](storage-use-azcopy-blobs.md)
```

Comment:
these don't appear to be valid links yet, are these supposed to point to the docs website?